src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -14,6 +14,8 @@ public class TorpedoStore {
 
   private int torpedoCount = 0;
 
+  private Random generator = new Random();
```

Comment:
Using pseudorandom generator is not safe, consider using SecureRandom
See: https://docs.oracle.com/javase/8/docs/api/java/security/SecureRandom.html

Reply:
Fixed later on Sonar Cloud, not an error per excersise